api/capi/include/nntrainer-tizen-internal.h

```diff
@@ -21,6 +22,21 @@
 extern "C" {
 #endif /* __cplusplus */
 
+/**
+ * @brief Get neural network layer from the model with the given name.
+ * @details Use this function to get already created Neural Network Layer. The
+ * returned layer must not be deleted as it is owned by the model.
```

Comment:
Add a note that this works only for layers added by the API, and does not currently support layers supported for layers created by the ini.
I will try and add it, but if not, its better to match the documentation to the current status.

Reply:
Okay :)

Reply:
is it possible to modify the layer which given by this API? I mean, let's say user changes dimension, unit size. then whole network need to be compiled. If it is the case we are considering here, then we need to add this description.  Or we are not allow to change, allow just get the information of that layer, then we need to add description for this purpose too.

Reply:
I think it's good point! I'll add description that modification after calling model_compile will have no effect.

Reply:
Great. Thanks.